cromulent/vocab.py

```diff
@@ -77,33 +77,43 @@ def register_instance(name, data):
+	"RightsStatement": {"parent": LinguisticObject, "id": "300435434", "label": "Copyright/License Statement", "brief": True},
+	"ValuationStatement": {"parent": LinguisticObject, "id": "300435426", "label": "Appraisal Statement", "brief": True},
+	"ConditionStatement": {"parent": LinguisticObject, "id": "300435425", "label": "Condition Statement", "brief": True},
+	"SignificanceStatement": {"parent": LinguisticObject, "id": "300435427", "label": "Significance Statement", "brief": True},
+	"ContextStatement": {"parent": LinguisticObject, "id": "300435428", "label": "Historical/Cultural Context", "brief": True},
+	"SourceStatement": {"parent": LinguisticObject, "id": "300404764", "label": "Source Statement", "brief": True},
+  
+	"MarkingsStatement": {"parent": LinguisticObject, "id": "300435420", "label": "Markings", "brief": True},
+	"InscriptionStatement": {"parent": LinguisticObject, "id": "300435414", "label": "Inscription", "brief": True},
+	"SignatureStatement": {"parent": LinguisticObject, "id": "300435415", "label": "Signature", "brief": True},
+	"WatermarksStatement": {"parent": LinguisticObject, "id": "300435421", "label": "Watermarks", "brief": True},
+	"MaterialStatement": {"parent": LinguisticObject, "id": "300435429", "label": "Material Statement", "brief": True},
+	"DimensionStatement": {"parent": LinguisticObject, "id": "300435430", "label": "Dimension Statement", "brief": True},
+	"CultureStatement": {"parent": LinguisticObject, "id": "300435431", "label": "Culture Statement", "brief": True},
+	"PeriodStatement": {"parent": LinguisticObject, "id": "300435432", "label": "Period Statement", "brief": True},
+
+	"ProductionStatement": {"parent": LinguisticObject, "id": "300435436", "label": "Production Statement", "brief": True },
+	"AcquisitionStatement": {"parent": LinguisticObject, "id": "300435439", "label": "Acquisition Statement", "brief": True},
+	"ProvenanceStatement": {"parent": LinguisticObject, "id":"300435438", "label": "Provenance Statement", "brief": True},
+	"BibliographyStatement": {"parent": LinguisticObject, "id": "300026497", "label": "Bibliography Statement", "brief": True},
+	"ExhibitionStatement": {"parent": LinguisticObject, "id":"300435424", "label":"Exhibition Statement", "brief": True},
+
+	"BiographyStatement": {"parent": LinguisticObject, "id":"300435422", "label": "Biography Statement", "brief": True},
+	"EditionStatement": {"parent": LinguisticObject, "id":"300435435", "label": "Edition Statement", "brief": True},
+	"PaginationStatement": {"parent": LinguisticObject, "id":"300435440", "label": "Pagination Statement", "brief": True},
+	"FoliationStatement": {"parent": LinguisticObject, "id":"300435441", "label": "Foliation Statement", "brief": True},
+	"LanguageStatement": {"parent": LinguisticObject, "id": "300435433", "label": "Language Statement", "brief": True},
+
+	"TranscribedInscription": {"parent": LinguisticObject, "id": "300028702", "label": "Transcribed Inscription", "brief": True},
```

Comment:
Hard to tell from this visual diff. Is this both a reorganization and an addition? Or are there also changes to existing classes?

Reply:
Sorry, I reordered into hopefully more obvious groups (description of the object, description of activity, description of non-object, transcription vs description) _and_ changed several of the ids to match Patricia's new terms.